### PR TITLE
MSA PR in the PR

### PR DESCRIPTION
**Pull request by aurel-l**
PR inside the PR

handles:
 - connect align label checkboxes to blast and align action buttons
   - add checkboxes and connect to action buttons in phylotree ad pim components
   - in wrapped view, toggle all checkboxes corresponding to the same sequence
 - floor the coordinate number that was appearing as a decimal when first switching to overview
 - make whole align label clickable as way to select one of the alignments
 - optimise use of grid and cell positionning
   - push the cells to the bottom to keep alignment without injecting dynamic style depending on track information
   - merge "rows" into one big section and handle grids as grids (stuff all the cells in one grid, and assign areas to those cells)
   - define size of side columns depending on content and not based on fixed sizing anymore -> better use of space

Notes and rough edges:
 - ~in wrapped view, the right column gets a different size based on the size of number displayed~
   - -> ~fixable by defining a huge grid with repeated grid-area patterns~ -> done (and it even simplifies the code)
 - in overview, the zoom of the nightingale components jumps every time the checkbox is selected

### CODE REVIEW

File: src/tools/components/Wrapped.tsx [hunk 1]
```diff
@@ -113,68 +113,52 @@ const MSAWrappedRow: FC<MSAWrappedRowProps> = ({
...
+          >
+            {s.name || ''}
+          </AlignLabel>
+        ))}
+      </div>
+      <div className="track">
+        <protvista-msa
+          ref={setMSAAttributes}
+          length={rowLength}
+          height={sequences.length * 20}
```
Comment (dlrice) [inline src/tools/components/Wrapped.tsx:143]: Could make use of `heightStyle` here

File: src/tools/components/Overview.tsx [hunk 2]
```diff
@@ -150,88 +153,77 @@ const AlignOverview: FC<BlastOverviewProps> = ({
...
+      <div className="track">
+        <protvista-manager
+          ref={managerRef}
+          attributes="displaystart displayend"
+        >
+          <protvista-navigation length={alignmentLength} />
+          <protvista-msa
+            ref={setMSAAttributes}
+            length={alignmentLength}
+            height={alignment.length * 20}
```
Comment (dlrice) [inline src/tools/components/Overview.tsx:207]: Could make use of `heightStyle` here

**Review by dlrice — COMMENTED** [review]
Is it possible to prevent the checkbox selection on an align sequence from changing which data populates the annotation track?

**Comment by aurel-l** [comment]
> Is it possible to prevent the checkbox selection on an align sequence from changing which data populates the annotation track?

Handled this and your inline comments in the latest commit

**Review by dlrice — APPROVED** [review]

**Pull request merged**